standard_renderer.go

```diff
@@ -483,10 +483,10 @@ type printLineMessage struct {
 	messageBody string
 }
 
-// Printf prints above the Program. This output is unmanaged by the program and
+// Println prints above the Program. This output is unmanaged by the program and
 // will persist across renders by the Program.
 //
-// Unlike fmt.Printf (but similar to log.Printf) the message will be print on
+// Unlike fmt.Println (but similar to log.Println) the message will be print on
```

Comment:
I think we can actually drop this part of the comment altogether because `fmt.Println`, `log.Println`, and `tea.Println` all print on their own lines.

Reply:
They finish with a line feed, but they wouldn't *start* on their own line, right? That's how I understood this comment anyway.

Reply:
I _think _ they all start on their own lines as well (unless I’m totally missing something, which is not unlikely)

Reply:
Oh, they don't: they'll just start printing at the current cursor position, wherever that may be. Example:

```go
fmt.Print("Hello ")
fmt.Println("World")
```

Reply:
TIL. I stand corrected!